generators/service-object-storage/templates/swift/instrumentation.swift

```diff
@@ -3,10 +3,9 @@ import CloudEnvironment
 import BluemixObjectStorage
 import Dispatch
 
-var objStorage: ObjectStorage!
-
-func initializeServiceObjectStorage() throws {
+func initializeServiceObjectStorage(cloudEnv: CloudEnv) throws -> ObjectStorage {
     // Load credentials for Object Storage using CloudEnvironment
+    let objStorage: ObjectStorage
```

Comment:
If you move this declaration below the guard you won't need the type annotation since it would be inferred by the assignment.

Reply:
done